docs/reference/api-conventions.asciidoc

```diff
@@ -59,6 +51,29 @@ NOTE: Single index APIs, such as the <<docs>> and
 <<indices-aliases,single-index `alias` APIs>>, do not support multi-target
 syntax.
 
+[[hidden-indices]]
+==== Hidden indices
+
+Indices that are configured to be hidden with the <<index-hidden,index.hidden>> setting are 
+excluded from mult-target queries by default. 
+To include hidden indices, you must specify the `expand_wildcards` parameter.
+
+The backing indices for data streams are hidden indices, 
+and some features like {ml} store information in hidden indices. 
+
+Global index templates that match all indices are not applied to hidden indices.
+
+[[system-indices]]
+==== System indices
+
+{es} modules and plugins can store configuration and state information in internal _system indices_. 
+You should not directly access or modify system indices 
+as they contain data essential to the operation of the system.
+They are excluded from multi-target requests by default. 
```

Comment:
This line should be removed, system index resolution won't change until 8.0, and after that it'll be disallowed completely (which is already covered below).